pdns/dnsdistdist/Makefile.am

```diff
@@ -289,7 +288,6 @@ dnsdist_LDADD = \
 	$(YAHTTP_LIBS) \
 	$(LIBSODIUM_LIBS) \
 	$(FSTRM_LIBS) \
-	$(SANITIZER_FLAGS) \
```

Comment:
I _think_ we use to set the sanitizer flags into `LDADD` because of thread sanitizer, but I can check later.